test/plugin/base_test.rb

```diff
@@ -526,16 +631,16 @@ def test_one_log_ec2
     verify_log_entries(1, EC2_PARAMS)
   end
 
-  def test_struct_payload_log
+  def test_json_payload_log
```

Comment:
Yeah, I found the `json_payload` name a bit too specific, too, especially when some of the tests are testing no json input. Just changed the test names as suggested.